### ISSUE TITLE
[Bug]: Autocomplete keyboard navigation doesn't work for Tailwind

### ISSUE THREAD

**Issue by stsrki** [Megabit/Blazorise#5463]
### Blazorise Version

1.5.1

### What Blazorise provider are you running on?

Tailwind

### Link to minimal reproduction or a simple code snippet

https://tailwinddemo.blazorise.com/tests/autocomplete

### Steps to reproduce

Type in the first Autocomplete to show the menu. Press the key up/down.

### What is expected?

The focused item should change.

### What is actually happening?

The focused  item does not change.

### What browsers do you see the problem on?

Chrome
